src/GeneralStateTestsFiller/stEIP1559/intrinsicGen.js

```diff
@@ -0,0 +1,218 @@
+#! /usr/bin/node
+
+const headerBoilerPlate = `
+# Check the intrinsic gas formula:
+#
+# 21000 + 16 * non-zero calldata bytes + 4 * zero calldata bytes +
+# 1900 * access list storage key count + 2400 * access list address count
+#
+# Do not edit this file directly, it was created by
+# tests/src/GeneralStateTestsFiller/stEIP1559/intrinsicGen.js
+
+
+intrinsic:
+
+  _info:
+    comment: Ori Pomerantz qbzzt1@gmail.com
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 0x20000
+    currentGasLimit: 0xFF112233445566
+    currentNumber: 1
+    currentTimestamp: 1000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+    currentBaseFee: 1000
+
+
+  pre:
+    cccccccccccccccccccccccccccccccccccccccc:
+      balance: '1000000000000000000'
+      code: |
+          :yul {
+              sstore(0, add(1,1))
+          }
+      nonce: '0'
+      storage: {}
+
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '1000000000000000000'
+      code: '0x'
+      nonce: 1
+      storage: {}
+
+
+  transaction:
+    nonce: 1
+    to: cccccccccccccccccccccccccccccccccccccccc
+    value:
+    - 0
+    secretKey: "45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8"
+    maxFeePerGas: 1000
+    maxPriorityFeePerGas: 100
+`
+
+
+// data for test cases
+const dataList = [
+    {gas:   0,     data: '0x' },
+    {gas:   4,     data: '0x00' },
+//  {gas:  16,     data: '0x00000000' },
```

Comment:
is this the test with 10k cases?

Reply:
No, the one with 1440 cases

Reply:
sure no need for expectSection transaction only flag?

Reply:
Do you mean `expectException`? That is already there, it just gets created later in the file. Or do you mean something else?

Reply:
I mean generate transactions that only covered with expectSection. to reduce the number of combinations

Reply:
I tried that, but when a combination is not covered in the expect section I get an error:
```
Error: Test has transaction uncovered with expect section! (stEIP1559/intrinsic, fork: all, TrInfo: d: 28, g: 55, v: 0)
```

Do you want me to do it anyway?